src/packages/cli/src/Init.ts

```diff
@@ -143,6 +144,11 @@ export class Init implements Command {
       url = args['--url']
     }
 
+    if (args['--provider'] && !args['--url']) {
+      provider = args['--provider']
+      url = `${provider}://johndoe:randompassword@localhost:5432/mydb?schema=public`
```

Comment:
We could differentiate the default port and existance of the schema (only on Postgres) as well.

Reply:
Nice one. Will add them..

Reply:
Please correct me if I'm wrong. I think you can query postgres to check for the existence of the schema only if you have the right credentials. It's very very rare to have the same credentials. I don't think it's worth it. I'd love to hear your thoughts on this.

Reply:
What I meant is that on MySQL you do not specify a `schema` as it does not do anything. Only on PostgreSQL you do.

Reply:
Ah, sorry. Got it.